tests/onebox.go

```diff
@@ -705,17 +705,6 @@ func (c *temporalImpl) overrideHistoryDynamicConfig(client *dcClient) {
 	client.OverrideValue(dynamicconfig.VisibilityProcessorUpdateAckInterval, 1*time.Second)
 }
 
-func (c *temporalImpl) RefreshNamespaceCache() {
-	c.frontendNamespaceRegistry.Refresh()
```

Comment:
you can remove the Refresh method and the whole triggerRefreshCh, I don't think anything else uses it